### PR TITLE
fix latest compiler

### PR DESCRIPTION
**Pull request by YiraSan**
there's a little miss

### CODE REVIEW

File: src/aio/posix/windows.zig [hunk 1]
```diff
@@ -60,7 +60,7 @@ pub const Iocp = struct {
         // > This completion key should be unique for each file handle, and it accompanies the file handle throughout the
         // > internal completion queuing process.
         id: u16,
-        _: @Type(.{ .Int = .{ .bits = @bitSizeOf(usize) - @bitSizeOf(u16) * 2, .signedness = .unsigned } }) = undefined,
+        _: @Type(.{ .@"int" = .{ .bits = @bitSizeOf(usize) - @bitSizeOf(u16) * 2, .signedness = .unsigned } }) = undefined,
```
Comment (Cloudef) [inline src/aio/posix/windows.zig:63]: I don't think this requires `@"int"`

Comment (YiraSan) [reply src/aio/posix/windows.zig]: you're totally right !

**Comment by Cloudef** [comment]
This won't fix windows builds due to the zigwin32 dep, but it's still required anyways.

**Comment by YiraSan** [comment]
https://github.com/kcbanner/zigwin32gen/commit/dc066ceda1ace1e1de7bb09b45f024f4cd0238c2

Also, why not using this fixing commit ?

**Comment by Cloudef** [comment]
Sure, you can add that to this pr

**Comment by YiraSan** [comment]
wait I have an issue in build.zig

**Comment by YiraSan** [comment]
I'm a big idiot, I used zigwin32gen instead of zigwin32 :')

**Comment by Cloudef** [comment]
There is no action checks for windows builds yet, so just send a new pr

**Pull request merged**